background-colors.suite/Scripts/randy2.script

```diff
@@ -0,0 +1,54 @@
+﻿//You know like an actual comment that tell you what the script is about
+
+put {
+	"Voting Certificates":"#EFDFE7", 
+	"Wards":"#E7CBD6",
+	Self:"#FFFFFF",
+	"Total Votable Shares":"#D6D7D6"
+} into bgColor
+put (562,208,881,368) into SearchArea
+
+// Can make into a function that takes in bgColor and SearchArea
+put the number of keys of bgColor into bgCount
+put keys(bgColor) into bgKeys
+repeat bgCount times
+	
+	put item repeatindex() of bgKeys into SingleBGKey
+	if imagefound(text:SingleBGKey) then
+		put FoundImageInfo().ImageRectangle into FiiR
+		put bgColor.(SingleBGKey) into TColor
+		put me.DefineColorArea (FiiR) into (CountColorMatrix,ColorMatrix)
+		if CountColorMatrix.(TColor) >=1
+			LogSuccess	"Found:"&& SingleBGKey && "with Pixel count"&&CountColorMatrix.(TColor) &&"Color" && TColor && "Within Area" && FiiR
```

Comment:
I'm not seeing any validation that the table values are listed in the proper vertical order. Let me know if I missed something.